.github/workflows/main.yaml

```diff
@@ -349,3 +349,44 @@ jobs:
         run: |
           export DOCKER_BUILD_AND_SYSTEM_PRUNE="true"
           make image 
+  
+  container_runtime_test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        container-runtime: ['containerd', 'cri-o', 'docker']
```

Comment:
`containerd` is no longer needed in this job, it has been verified in e2e_test job.

Reply:
You mean, job will be like this:
![image](https://github.com/kubeedge/kubeedge/assets/98945276/0141a6cf-9f84-46a2-a3f1-fce993f49eb7)
right?

Reply:
> You mean, job will be like this: ![image](https://user-images.githubusercontent.com/98945276/285365596-0141a6cf-9f84-46a2-a3f1-fce993f49eb7.png) right?

yes

Reply:
✅